src/blocks/content-helper/components/related-top-post-list.tsx

```diff
@@ -48,6 +48,8 @@ function RelatedTopPostList() {
 					} else {
 						setLoading( false );
 						setError( err );
+
+						throw err;
```

Comment:
I think we should throw the error even after catching it so that the browser console can shows the error instead of showing nothing.

Reply:
Not a good idea for now because we are showing error messages based on this.